ola.spec.in

```diff
@@ -107,21 +103,15 @@ cp debian/ola.udev %buildroot/%{_udevrulesdir}
 %{_libdir}/pkgconfig/*
 
 %files -n python2-%{name}
-%{python2_sitelib}/ola
```

Comment:
This seems rather inconsistent, wouldn't it be better to favour either *.py, or the whole folder, rather than using a mix of both?